### PR TITLE
Some fixes by ramin

### PR DESCRIPTION
**Pull request by RamRamez**
1. add device size to design system

2. fix H5 and H6 font size on mobile according to figma

3. fix button loader color when disabled

4. fix primary button color when disabled

### CODE REVIEW

File: src/components/buttons/OulineButton.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const ButtonContainer = styled.button<IButtonContainerProps>`
 		switch (props.buttonType) {
 			case 'primary':
 				return props.disabled
-					? `color: ${brandColors.giv[500]};border-color: ${brandColors.giv[500]};opacity: 0.4;`
+					? `color: ${brandColors.pinky[300]};border-color: ${brandColors.pinky[300]};`
```
Comment (MohammadPCh) [inline src/components/buttons/OulineButton.tsx:24]: Is it based on the design?

File: src/components/buttons/Button.tsx [hunk 2]
```diff
@@ -21,6 +21,9 @@ const ButtonContainer = styled.button<IButtonContainerProps>`
 			if (props.buttonType === 'texty') {
 				return `color: ${brandColors.giv[500]};background: unset};padding: 8px 24px;opacity: 0.5;`;
 			}
+			if (props.buttonType === 'primary') {
+				return `background: ${brandColors.pinky[300]};color: ${brandColors.pinky[200]};`;
+			}
```
Comment (MohammadPCh) [inline src/components/buttons/Button.tsx:26]: Is it based on the design?

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
@RamRamez I think the changes are not based on the design.

**Comment by RamRamez** [comment]
> @RamRamez I think the changes are not based on the design.

Yeah, I talked with @mosaeedi about these

File: src/components/buttons/OulineButton.tsx (hunk 1 above)
Comment (RamRamez) [reply src/components/buttons/OulineButton.tsx:24]: when button is pinky, it's obvious that its disabled state should be something like pinky not purple. my changes are for primary theme.

File: src/components/buttons/Button.tsx (hunk 2 above)
Comment (RamRamez) [reply src/components/buttons/Button.tsx:26]: same as the above

**Review by MohammadPCh — APPROVED** [review]
Thanks @RamRamez

**Pull request merged**